subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/DependencyReportTask.java

```diff
@@ -26,6 +27,7 @@
 public abstract class DependencyReportTask extends AbstractDependencyReportTask {
 
     @Override
+    @ToBeKeptEagerProperty(because = "ConfigurationContainer is injected type")
     public ConfigurationContainer getTaskConfigurations() {
```

Comment:
❗ Should not need annotation as a `DOC`.

Reply:
Done